SwiftReflector/SwiftXmlReflection/BaseDeclaration.cs

```diff
@@ -358,16 +358,42 @@ public bool IsTypeSpecGenericReference (TypeSpec sp)
 			return ns != null && IsTypeSpecGeneric (ns.Name);
 		}
 
-		public bool IsTypeSpecGeneric (string typeSpecName)
+		public bool IsTypeSpecGenericMetatypeReference (TypeSpec sp)
+		{
+			if (sp.ContainsGenericParameters)
+				return false;
+			string notUsed;
+			return sp is NamedTypeSpec ns && IsTypeSpecGenericMetatypeReference (ns.Name, out notUsed);
```

Comment:
You can use `var _` for out variables you don't need:

```suggestion
			return sp is NamedTypeSpec ns && IsTypeSpecGenericMetatypeReference (ns.Name, out var _);
```

Reply:
TIL